### ISSUE TITLE
FPS drops when in first person while inside a ship

### ISSUE THREAD

**Issue by FergorH** [tsunamayo/Starship-EVO#5019]
### Short Description of the issue:
I noticed my FPS drops by 20 frames, give or take, when i'm in first person inside a ship. Let be noted i only noticed this inside the tutorial repair ship, i haven't tried it on other ships. I tried taking a video of it but my FPS counter didn't appear, so this is gonna be a "source: trust me bro" situation.

...

### Environment
- Starship EVO version: 22w50b
- Graphics Card: GTX 1050

Player.log
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/10230646/Player.log)

**Comment by tsunamayo** [comment]
@FergorH Hi, could you please tell me more about that issue? I cant reproduce on my side.
- What is your CPU and monitor res and refresh rate
- What is you normal FPS level? 
Thanks

**Comment by FergorH** [comment]
Of course. I've since tried it on other ships and have seen the same thing. If i am seated in the cockpit (just seated is enough, not necessarily controlling the ship yet) and i am playing in first person my FPS hovers around 50-60, give or take, out in open space, but the moment i press F4 and go onto the third person camera my FPS shoots up to 70-90, again, while out in open space. 

* I have an 8th gen i5-8300H, my monitor is 1080p (Though i usually lower the game's resolution to 720p to get more performance) and the refresh rate is 60hz i believe.
* My FPS varies quite a bit, out in open space it hovers around 60-70 if i'm playing in 1080p, in 720p it's around 80-90. Being around massive creations really affects it, the stations in particular really tanks it, down to 30s. The rings are fine, usually high 40s to 60, sometimes 70.

**Comment by tsunamayo** [comment]
Last question: was it is like this since you started playing the game or is it like that since recently?

I am not sure there is a bug at all, of course if you are GPU limited and there is nothing to show in the world your FPS will be high. When there is a ship / station to draw then it will tank, but I would say it is completely normal. Achieving 60-70 on a 1050 in 1080p seems ambitious...

Thanks

**Comment by FergorH** [comment]
I am honestly not sure unfortunately, it's quite possible it was always like that and i only noticed it now. I just thought it was quite the random thing to happen, when i mentioned it on discord they told me to create an issue for it.

Oh of course, i know my specs aren't the best, i'm really not complaining when it comes to performance, it is quite playable for me. Fps gain on third person, i just found it weird, maybe it is because the pieces in the cockpit around the character are rendered when in first person? i really don't know.
